main_test.go

```diff
@@ -37,12 +37,40 @@ func Test_getContributingURL(t *testing.T) {
 	}
 }
 
-func Test_hmacValidationDefault(t *testing.T) {
-	os.Setenv("hmac_validation", "")
-	enabled := hmacValidation()
-	want := false
-	if enabled != want {
-		t.Errorf("want %t got %t", want, enabled)
-		t.Fail()
+func Test_customerValidation(t *testing.T) {
+	tests := []struct {
+		title        string
+		value        string
+		expectedBool bool
+	}{
+		{
+			title:        "`validate_hmac` is unset",
```

Comment:
How about? ``validate_hmac` is unset, defaults to on`

Reply:
Sounds good ^